### DISCUSSION TITLE
Contributor call: 16 April 2022

### DISCUSSION THREAD

**Discussion by joostdecock** [freesewing/freesewing#2058] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lexander2002 on our Discord server.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 16 April 2022 5pm UTC, which is can be found in your local time zone [here](https://savvytime.com/converter/utc-to-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/Mar-19-2022/5pm)
Or check the bot in Discord! #meetings
**⚠Important note:** timezones have been switching up lately. 
The meetings will always be at 6pm, Antwerp time.

### Agenda

- Todo. Suggestions welcome

> This discussion will be updated after the call with notes and to-dos.

**Comment by bobgeorgethe3rd** [comment]
- Logistics and what needs to be done to add @TriploidTree 's idea of adding new pages "Maker's Notes" and "Faqs" to the documentation.
- Discussing a possibily of having a Maintainers Directory either a full list as a discussion and/or a Read Me File in the Patterns Packages.
  - Pattern Code
  - Pattern Docs
  - Website Docs
  - etc.

**Comment by l3xander** [comment, marked as answer]
### Notes
- @bobgeorgethe3rd brought up @triploidtree’s idea of adding new pages to patterns, “Maker’s Notes” and “Faqs”, but we will discuss that when she’s on the call.
- @bobgeorgethe3rd suggested making a maintainer’s directory that would look [something like this](https://cdn.discordapp.com/attachments/788066844393078835/964921704763760741/unknown.png), so we know who to refer to and if someone needs to take something over.
- @tangerineshark on ideas for the next newsletter.
    - As for interviews, we have one left of someone who’s currently inactive, so we are thinking of doing another one in the meantime. There are a lot of people who we can interview still.
        - @joostdecock is a candidate for an interview, but he suggested to only do that with a big FreeSewing update.
        - I (@lexander2002) mentioned that I have submitted a talk to [MCH 2022](https://mch2022.org/#/). If that goes through it will be held this summer, so that would be relevant for the next interview. @eleonoremilbou added that it would also encourage people to look into joining the event.
        - @eleonoremilbou offered to interview @tangerineshark in the future.
    - @joostdecock about the October issue: Simon J from the Discord will do a pop-up in London of their project, an on-demand pattern factory based on FreeSewing’s patterns. This would make for an interesting article.
    - @joostdecock suggested an article on including people with disabilities, as patterns work differently in those cases (e.g. making patterns suitable for wheelchair users). He wants to actively search for people who know about these issues and can design for it. @eleonoremilbou mentioned she knows someone on Instagram she could ask.
- @bobgeorgethe3rd is working on his Billie the Bucket hat pattern, the instructions are better now.
- @joostdecock updated us that he is working on the known armhole issue in Brian which extends into other patterns, but it takes time not to break everything.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
